proto/src/determined/trial/v1/trial.proto

```diff
@@ -195,7 +195,7 @@ message TrialEarlyExit {
 }
 
 // Metrics from the trial some duration of training.
-message TrialMetrics {
+message TrialMetrics {  // SEE MetricsReport
```

Comment:
why see MetricsReport?

Reply:
they're very similar but subtly different. I was gonna see if they needed to be different after this project. and in either case, it might make sense to link these two somehow (even if it's just via a mention)